src/utils/deviceInfo/tests/deepDeviceInfo.test.ts

```diff
@@ -74,6 +74,16 @@ describe('A browser that doesnt support userAgentData', () => {
         expect(deviceInfo.deviceType).toBe('desktop');
         expect(deviceInfo.screenDimensions).toBeDefined();
     });
+
+    test('Simulated Samsung device running Samsung Internet', () => {
+        const mockedUserAgent =
```

Comment:
Consistency: Could you add newlines, just to group the different parts of the test (in the similar manner as the other tests in this file)?